config/templates/cloud-config-controller

```diff
@@ -344,7 +344,7 @@ write_files:
           - --tls-private-key-file=/etc/kubernetes/ssl/apiserver-key.pem
           - --client-ca-file=/etc/kubernetes/ssl/ca.pem
           - --service-account-key-file=/etc/kubernetes/ssl/apiserver-key.pem
-          - --runtime-config=extensions/v1beta1/networkpolicies=true
+          - --runtime-config=extensions/v1beta1/networkpolicies=true, batch/v2alpha1
```

Comment:
@danielfm Works both ways and actually I have it like this `batch/v2alpha1=true`. Thought using the recommended method from the documentation `--runtime-config=batch/v2alpha1`